reya_data_feed/resources/wallet.py

```diff
@@ -0,0 +1,254 @@
+        
+        Args:
+            socket: The WebSocket connection to use for this resource.
+        """
+        super().__init__(socket, "/api/trading/wallet/{address}/accounts/balances")
+    
+    def for_wallet(self, address: str) -> 'WalletBalancesSubscription':
+        """Create a subscription for a specific wallet's account balances.
+        
+        Args:
+            address: The wallet address.
+            
+        Returns:
+            A subscription object for the wallet balances.
+        """
+        return WalletBalancesSubscription(self.socket, address)
+
+class WalletPositionsSubscription:
+    """Manages a subscription to positions for a specific wallet."""
+    
+    def __init__(self, socket: 'ReyaSocket', address: str):
+        """Initialize a wallet positions subscription.
+        
+        Args:
+            socket: The WebSocket connection to use for this subscription.
+            address: The wallet address.
+        """
+        self.socket = socket
+        self.address = address
+        self.path = f"/api/trading/wallet/{address}/positions"
+    
+    def subscribe(self, batched: bool = False) -> None:
+        """Subscribe to wallet positions.
+        
+        Args:
+            batched: Whether to receive updates in batches.
+        """
+        self.socket.send_subscribe(channel=self.path, batched=batched)
+    
+    def unsubscribe(self) -> None:
+        """Unsubscribe from wallet positions."""
+        self.socket.send_unsubscribe(channel=self.path)
+
+class WalletOrdersSubscription:
+    """Manages a subscription to orders for a specific wallet."""
+    
+    def __init__(self, socket: 'ReyaSocket', address: str):
+        """Initialize a wallet orders subscription.
+        
+        Args:
+            socket: The WebSocket connection to use for this subscription.
+            address: The wallet address.
+        """
+        self.socket = socket
+        self.address = address
+        self.path = f"/api/trading/wallet/{address}/orders"
+    
+    def subscribe(self, batched: bool = False) -> None:
+        """Subscribe to wallet orders.
+        
+        Args:
+            batched: Whether to receive updates in batches.
+        """
+        self.socket.send_subscribe(channel=self.path, batched=batched)
+    
+    def unsubscribe(self) -> None:
+        """Unsubscribe from wallet orders."""
+        self.socket.send_unsubscribe(channel=self.path)
+
+class WalletBalancesSubscription:
+    """Manages a subscription to account balances for a specific wallet."""
+    
+    def __init__(self, socket: 'ReyaSocket', address: str):
+        """Initialize a wallet balances subscription.
+        
+        Args:
+            socket: The WebSocket connection to use for this subscription.
+            address: The wallet address.
+        """
+        self.socket = socket
+        self.address = address
+        self.path = f"/api/trading/wallet/{address}/accounts/balances"
```

Comment:
one thing that i like about this account_update endpoint from binance is that they combine all account updates (position and balances) into a single channel and within that for each update they have a reason for that update, e.g. if it was caused by a transfer or deposit or order execution

i think we should be fine keeping position and balance updates as separate channels for now, but for balances it'd be nice to also have the reason for a balance change (we don't expose it in a clean way currently through an on-chain event, smth we should check with @ioanaroceanu2000)

dropping this thought here to not forget, doesn't apply to this PR

https://developers.binance.com/docs/derivatives/usds-margined-futures/user-data-streams/Event-Balance-and-Position-Update